homeassistant/components/modbus/base_platform.py

```diff
@@ -92,11 +94,11 @@ async def async_update(self, now=None):
         """Virtual function to be overwritten."""
 
     @callback
-    def async_remote_start(self) -> None:
+    def async_run(self) -> None:
         """Remote start entity."""
-        if self._cancel_timer:
-            self._cancel_timer()
-            self._cancel_timer = None
+        self.async_hold(update=False)
+        if self._scan_interval == 0 | self._scan_interval > ACTIVE_SCAN_INTERVAL:
```

Comment:
Did you intent to do binary OR ?
```suggestion
        if self._scan_interval == 0 or self._scan_interval > ACTIVE_SCAN_INTERVAL:
```

Reply:
my bad I was doing a C++ patch in parallel.